packages/docusaurus-types/src/index.d.ts

```diff
@@ -104,6 +110,11 @@ export interface Plugin<T> {
   getPathsToWatch?(): string[];
   getClientModules?(): string[];
   extendCli?(cli: Command): void;
+  injectHtmlTags?(): {
+    headTags?: string | HtmlTagObject | (string | HtmlTagObject)[];
```

Comment:
How about put this to a separate type?

```ts
type HtmlTags = string | HtmlTagObject | (string | HtmlTagObject)[];
```

And rename the `HtmlTags` interface to something like `InjectedHtmlTags` (considering pre/post tags, this was even more clear).

Reply:
yes thats a good idea. I am currently out of reach from my pc but if you want to help me edit the PR thats also very appreciated :)